src/core/lib/debug/stats_data.yaml

```diff
@@ -159,6 +159,8 @@
   doc: Number of HTTP2 writes initiated due to 'ping_response'
 - counter: http2_initiate_write_due_to_force_rst_stream
   doc: Number of HTTP2 writes initiated due to 'force_rst_stream'
+- counter: http2_spurious_writes_begun
```

Comment:
You can remove the http2_initiate_write_due_to_bdp_estimator_ping counter

Reply:
I have a couple of bugs to sort out in schema generation first...